packages/es-components/src/components/containers/popover/Popover.js

```diff
@@ -281,7 +281,8 @@ Popover.defaultProps = {
   title: undefined,
   enableEvents: true,
   strategy: 'absolute',
-  keepTogether: true
+  keepTogether: true,
+  styleType: 'primary'
```

Comment:
Looks like you need to add the default for `popoverWrapperClassName`, which can just be `undefined`.

Reply:
It's probably fine but I'm guessing most downstream users aren't going to utilize it -- so using `undefined` is a bit cleaner.

Reply:
Interesting.. I thought it was a bit odd to see 'undefined' in my HTML inspector. I've pushed it up though. Sorry I didn't run the linter before! And thanks for the review.